content/future.html

```diff
@@ -2,104 +2,299 @@ <h1 id="future"> Future </h1>
+  thus Canadian companies often skip this acceditation and hire employees
+  as "Software Developers".
+</p>
+<div class="half second">
+  <span class="graph-title">Will Pursue P.Eng Accreditation</span>
+  <div id="peng"></div>
+</div>
+
+<div class="vbx3"></div>
+
+<p class="half">
+  <span class="big">
+    Students believe in the value of their FYDP projects. 
+  </span>
+  More than half of respondents may or will continue work on their FYDP.
+  Hopefully this shows that students have selected projects in areas they
+  are interested in or that they believe will be beneficial to pursue
+  further.
+</p>
+<div class="half second">
+  <span class="graph-title">Will Continue FYDP Work</span>
+  <div id="cont-fydp"></div>
+</div>
+
+<div class="vbx3"></div>
+
+<p class="half">
+  <span class="big">
+    Students are very content with their post-grad plans with 52% feeling
+    they are perfect and 91% rating their future a 4/5 or higher.
+  </span>
+  Given the state of the world at this moment, it is reassuring to see
+  SE students being satisfied with their situations.
+</p>
+<div class="half second">
+  <span class="graph-title">Post-grad contentness</span>
+  <div id="post-contentness"></div>
+</div>
+
+<div class="vbx2"></div>
+<h3> Full-Time Compensation </h3>
+<p class="half">
+  <span class="big">
+    The compensation for the first-year of full-time work for respondents
+    will average $284,728 CAD, with a median of $260,000.
   </span>
-  This could explain why California was a popular choice for full-time location.
+  The exchange rate used for this question and the ones that follow was 1.3
+  CAD per 1 USD. That rate was accurate at the time the survey was shared,
+  but the rate is 1.2 as of writing.
   <br /><br />
-  Although family may be the least popular factor, most students working away
-  from home have expressed interest in working near their families in the
-  future.
+  Since 77% of respondents will be relocating to the US, we can estimate
+  the new average and median earnings to actually be $267,882 and $244,600 CAD.
+  <!-- 260k*(1 + 0.77(1.2/1.3 - 1)). -->
 </p>
 <div class="half second">
-  <span class="graph-title">Factors affecting post-graduation plans</span>
-  <div id="motivation"></div>
+  <span class="graph-title">New Grad Total First-Year Compensation</span>
+  <div id="ft-total"></div>
 </div>
 
 <div class="vbx2"></div>
+<p class="half">
+  <span class="big">
+    The hourly wage / salary will have a median and an average of $81 and
+    $105 CAD respectively.
+  </span>
+  We can again estimate this to be $76 and $99 CAD with today's exchange rate.
+</p>
+<div class="half second">
+  <span class="graph-title">New Grad Hourly Wage</span>
+  <div id="ft-hourly"></div>
+</div>
 
-<h3> Debt </h3>
+<div class="vbx2"></div>
 <p class="half">
   <span class="big">
-    65% of students are graduating debt-free.
+    There's a huge discrepency in profit sharing for new grads,
```

Comment:
What do you mean by porfit sharing? You mean stock options?

Reply:
yes, profit sharing is a general term I'm using because some companies might give options vs. others might give RSUs